### PR TITLE
Implement 2-pin radial and axial landpatterns 

### PR DESCRIPTION
**Pull request by m-hilgendorf**

### CODE REVIEW

**Comment by bpellenard** [comment]
I would recommend to create a separate PR for the change in .github/workflows/test.yml

File: tests/test-th-land-patterns.stanza [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    )
+  if p?:
+    pin a
+    pin c
+    landpattern = lp(a => lp.a, c => lp.c)
+  else:
+    port p: pin[[1 2]]
+    landpattern = lp(p[1] => lp.p[1], p[2] => lp.p[2])
+  make-box-symbol()
+pcb-component axial (r:String, p?:True|False):
```
Comment (bpellenard) [inline tests/test-th-land-patterns.stanza:30]: what is representing `p?`

Comment (m-hilgendorf) [reply tests/test-th-land-patterns.stanza:30]: r is the reference designator and p? Is if it is polarized. I can rename, but these are not public facing

**Review by bpellenard — APPROVED** [review]
Looks good to me!

**Pull request merged**